### PR TITLE
New package: idris-1.3.2

### PR DESCRIPTION
**Pull request by zhengqunkoo**
Regarding quality requirements: [`Idris2`](https://github.com/edwinb/Idris2), when packaged, will require `Idris`.

Unfortunately, building `Idris` via `haskell-stack` excludes the `Builtins` and `Prelude` modules, which means no REPL and no compilation of `Idris2` is possible.

### CODE REVIEW

File: srcpkgs/idris/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'idris'
+pkgname=idris
+version=1.3.2
+revision=1
+build_style=haskell-stack
+hostmakedepends="pkg-config ghc stack"
+makedepends="zlib-devel ncurses-devel"
+short_desc="Functional Programming Language with Dependent Types"
+maintainer="Zhengqun Koo <root@zhengqunkoo.com>"
+license="custom"
```
Comment (abenson) [inline srcpkgs/idris/template]: license appears to be `BSD-3-Clause`

Comment (zhengqunkoo) [reply srcpkgs/idris/template]: Thanks! It indeed is.

**Comment by zhengqunkoo** [comment]
All failed checks begin with:
```
/usr/bin/ld: /usr/lib/gcc/x86_64-linux-musl/9.2.0/../../../../lib/crt1.o: in function `_start_c':

/builddir/musl-1.1.23/crt/crt1.c:18:0: error:

     undefined reference to `main'
```

As far as I know, this is caused by missing ghc or missing stack hostmakedependencies.

**Comment by abenson** [comment]
Seemingly abandoned.

**Pull request closed without merging**